tesseract-clickhouse/src/sql/primary_agg.rs

```diff
@@ -40,7 +41,13 @@ pub fn primary_agg(
     // So just swap the primary key DimSubquery to the head
 
     let mut ext_drills: Vec<_> = drills.iter()
-        .filter(|d| d.table.name != table.name)
+        .filter(|d| {
```

Comment:
hmmm... I feel like all inline tables should be external drills. If the inline table alias is the same as the fact table name, I would expect either
1) inline table should be further aliase (perhaps append `inline_table` or something)
2) there's an error during reading the schema, to make sure the names are not the same.